charts/opentelemetry-demo/examples/bring-your-own-observability/rendered/opensearch/poddisruptionbudget.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # Source: opentelemetry-demo/charts/opensearch/templates/poddisruptionbudget.yaml
-apiVersion: policy/v1
+apiVersion: policy/v1beta1
```

Comment:
this change is peculiar. It seems like your local version of kubectl is for [K8s version 1.21 or earlier](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/templates/poddisruptionbudget.yaml#L2-L6)

Upgrading your local kubectl / helm should fix this, and allow for the linter to pass.

Reply:
good catch, have addressed the problem, but what's weird is my kubectl version is GitVersion:"v1.27.11", have also tried with 1.29 but still the same
And was only able to build it without v1beta on a different host